server/app/controllers/admin/AdminApplicationController.java

```diff
@@ -282,7 +282,7 @@ public Result index(
     var paginationSpec = new PageNumberBasedPaginationSpec(PAGE_SIZE, page.orElse(1));
     PaginationResult<Application> applications =
         programService.getSubmittedProgramApplicationsAllVersions(
-            programId, F.Either.Right(paginationSpec), search);
+            programId, F.Either.Right(paginationSpec), search, TimeFilter.builder().build());
```

Comment:
suggestion (non-blocking): add `public static final EMPTY = TimeFilter.builder().build();` to `TimeFilter` and use that

Reply:
Done.